src/common/PasswordInput.js

```diff
@@ -16,11 +16,19 @@ const styles = createStyleSheet({
     top: 0,
     bottom: 0,
     justifyContent: 'center',
+    borderTopWidth: 1,
+    borderColor: BORDER_COLOR,
+    borderBottomWidth: 1,
+    borderRightWidth: 1,
   },
   showPasswordButtonText: {
     margin: 8,
     color: BRAND_COLOR,
   },
+  passwordTextInput: {
+    paddingRight: '15%',
```

Comment:
Hmm, I'd rather not use a percentage here. We should reserve exactly the right amount of space for the show/hide button, and this will cause that space to be too large or small depending on factors including:
- How wide the input is (e.g., varies with screen width; will be large on a tablet)
- How big the show/hide button's text is (varies with the user's chosen font size in system settings)